utils/debug/PointCloudDebug.js

```diff
@@ -0,0 +1,192 @@
+/* global itowns */
+function isInHierarchy(elt, hierarchyNode) {
+    if (elt.name.length > hierarchyNode.length) {
+        return elt.name.startsWith(hierarchyNode);
+    } else if (elt.name.length < hierarchyNode.length) {
+        return hierarchyNode.startsWith(elt.name);
+    } else {
+        return hierarchyNode === elt.name;
+    }
+}
+
+function PointCloudDebug(view, datUi) {
```

Comment:
Actually I'm wondering why we are exposing a constructor, as we don't expose any API OOP style. Maybe we can expose a `generatePointCloudDebugTools` instead. (it would prevent the no-new above too).